athena-elasticsearch/src/main/java/com/amazonaws/athena/connectors/elasticsearch/ElasticsearchRecordHandler.java

```diff
@@ -136,8 +140,18 @@ protected void readWithConstraint(BlockSpiller spiller, ReadRecordsRequest recor
         logger.info("readWithConstraint - enter - Domain: {}, Index: {}, Mapping: {}",
```

Comment:
this needs to be moved after the if statement; so we can log correctly for QPT and non-QPT.

Moreover; we need to change it to take `domain` `query` etc.

Reply:
Incorporated the required changes.